src/app/content/highlights/components/ColorIndicator.tsx

```diff
@@ -20,21 +20,44 @@ const CheckIcon = styled(Check)`
   display: none;
 `;
 
+interface FocusedStyleProps extends Pick<Props, 'style' | 'size' | 'shape'> {}
```

Comment:
Making
```
interface FocusedStyleProps {
  style: typeof highlightStyles[number];
  size?: 'small'
  shape?: 'square' | 'circle';
}
```
and then `Props` to extend it feels more clear, but I think it's debatable and it doesn't really matter. :P

Reply:
Btw. should `size` and `shape` be optional?

Reply:
I've did what you've suggested but I've changed the name of the interface to `StyleProps` so it makes more sense he they are separated from the `Props`

Reply:
> Btw. should size and shape be optional?

There is support for `undefined` values and this PR is not really doing anything about these values so I think yes, they should stay optional